pkg/kopia/snapshot/snapshot.go

```diff
@@ -21,16 +21,15 @@ import (
 	"strings"
 	"time"
 
+	"github.com/kanisterio/errkit"
+	"github.com/kanisterio/kanister/pkg/kopia"
+	"github.com/kanisterio/kanister/pkg/kopia/repository"
 	"github.com/kopia/kopia/fs"
 	"github.com/kopia/kopia/repo"
 	"github.com/kopia/kopia/repo/manifest"
 	"github.com/kopia/kopia/snapshot"
 	"github.com/kopia/kopia/snapshot/policy"
 	"github.com/kopia/kopia/snapshot/snapshotfs"
-	"github.com/pkg/errors"
-
-	"github.com/kanisterio/kanister/pkg/kopia"
-	"github.com/kanisterio/kanister/pkg/kopia/repository"
```

Comment:
`github.com/kanisterio/kanister` are internal packages, so should be kept in a separate group